eadf/importers.py

```diff
@@ -186,3 +189,53 @@ def fromAngleListData(
     )
 
     return EADF(arrInter, arrAzi, arrCoEle, arrFreqData, arrPos)
+
+def loadData(
+    path: str,
+    numPor = 0,
+) -> EADF:
+    """
+    This format defines angular data over uniformly sampled
+    ele, azi, elem, pol, and freq. Using the importer it is possible to choose
+    the number of ports to derive the EADF by supplying the numPor value.
```

Comment:
why must it be uniformly sampled along frequency and polarisation? what does that even mean? also, there is no description about the internal structure of the expected matfile. how would someone be able to produce a .mat-file that can be processed by this importer? however, this is crucially necessary.

Reply:
by this did you mean that the details about the measurement method itself are lacking?